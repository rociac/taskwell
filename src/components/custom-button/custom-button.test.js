import CustomButton from './custom-button.component';
import renderer from 'react-test-renderer';

describe('Custom Button', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<CustomButton>Button</CustomButton>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});