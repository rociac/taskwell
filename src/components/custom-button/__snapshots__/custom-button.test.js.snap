// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Button renders correctly 1`] = `
<button>
  Button
</button>
`;
